tls/s2n_x509_validator.c

```diff
@@ -655,19 +672,17 @@ S2N_RESULT s2n_validate_sig_scheme_supported(struct s2n_connection *conn, X509 *
 
     int nid = 0;
 
-    #if defined(LIBRESSL_VERSION_NUMBER) && (LIBRESSL_VERSION_NUMBER < 0x02070000f)
-        RESULT_ENSURE_REF(x509_cert->sig_alg);
-        nid = OBJ_obj2nid(x509_cert->sig_alg->algorithm);
-    #else
-        nid = X509_get_signature_nid(x509_cert);
-    #endif
+#if defined(LIBRESSL_VERSION_NUMBER) && (LIBRESSL_VERSION_NUMBER < 0x02070000f)
+    RESULT_ENSURE_REF(x509_cert->sig_alg);
+    nid = OBJ_obj2nid(x509_cert->sig_alg->algorithm);
+#else
+    nid = X509_get_signature_nid(x509_cert);
+#endif
 
     for (size_t i = 0; i < cert_sig_preferences->count; i++) {
-
         if (cert_sig_preferences->signature_schemes[i]->libcrypto_nid == nid) {
             /* SHA-1 algorithms are not supported in certificate signatures in TLS1.3 */
-            RESULT_ENSURE(!(conn->actual_protocol_version >= S2N_TLS13 &&
-                    cert_sig_preferences->signature_schemes[i]->hash_alg == S2N_HASH_SHA1), S2N_ERR_CERT_UNTRUSTED);
+            RESULT_ENSURE(!(conn->actual_protocol_version >= S2N_TLS13 && cert_sig_preferences->signature_schemes[i]->hash_alg == S2N_HASH_SHA1), S2N_ERR_CERT_UNTRUSTED);
 
```

Comment:
Too long

Reply:
I had a conversation with @harrisonkaiser about this and dont think it needs to be a blocker.

Since we dont enforce a line length check, clang doesnt care and this is something we can fix on an ongoing basis as we touch code. The auto format and actually enabling clang-format seem most important for these PRs.

The alternative is that we do fix it all in this PR now and that a bit of extra lift for Harrison.

Ok with either philosphy but enabling clang-format quicker seems more important to me

Reply:
Sure. `awk '{ if (length > 150) print FILENAME ":" FNR " " length " " $0 }' s2n_[s-z]*`

The remaining long lines (i.e. >150) are security policies.